class IOErrorException(Exception):
    pass

class RuntimeException(Exception):
    pass
